addons/crm/data/crm_lead_prediction_data.xml

```diff
@@ -20,6 +20,15 @@
         <record id="frequency_field_lang_id" model="crm.lead.scoring.frequency.field">
             <field name="field_id" eval="ref('crm.field_crm_lead__lang_id')"/>
         </record>
+        <function model="ir.config_parameter" name="set_param" eval="('crm.pls_fields', 'State,Country,Phone Quality,Email Quality,Source')"/>
```

Comment:
Does that mean that changing a field string breaks the pls config ?

Shoudln't it be using technical name (state_id,country_id, ...) ? Like in test_crm_pls ?

Anyway isn't pls an option ? People cannot not avoid it ?

cc @dbeguin

Reply:
indeed, this should not be based on fields string, I don't think this should either be based on technical_name but field_id. 
You should set the res.config.settings.predictive_lead_scoring_fields with the id of crm.lead.scoring.frequency.field.
Didn't try but should look like :
```
<record id="frequency_field_lang_id" model="res.config.settings">
    <field name="predictive_lead_scoring_fields" eval="[6,0, [
ref('crm.frequency_field_lang_id'), ref('crm.frequency_field_source_id'), etc..]"/>
</record>
```
But, not sure if this wasn't intended to not set the optional PLS fields as active. The idea was to have the closest behaviour of what existed before (only the stage should make an impact, and if you choose to active some optional fields, you can but on limited fields). See this with Nathan (NGR)

Reply:
This will be done in another task : https://www.odoo.com/web#id=2028958&action=333&active_id=965&model=project.task&view_type=form&menu_id=4720

I removed it.